beacon_node/lighthouse_network/src/types/globals.rs

```diff
@@ -110,6 +110,12 @@ impl<E: EthSpec> NetworkGlobals<E> {
         std::mem::replace(&mut *self.sync_state.write(), new_state)
     }
 
+    /// Compute custody data columns the node is assigned to custody.
+    pub fn custody_columns(&self, _spec: &ChainSpec) -> Vec<ColumnIndex> {
+        let _enr = self.local_enr();
+        todo!("implement ENR changes");
```

Comment:
```suggestion
        //TODO(das): implement ENR changes
        vec![]
```

Reply:
Fixed in 96e2363d721f55d79f66f98f15fd61a854c7dc62